e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/dashboard-deals-filter-by-status.spec.js

```diff
@@ -4,22 +4,31 @@ const CONSTANTS = require('../../../../../fixtures/constants');
 const CONTENT_STRINGS = require('../../../../../fixtures/content-strings');
 const { dashboardDeals } = require('../../../../pages');
 const { dashboardFilters } = require('../../../../partials');
-const { BSS_DEAL_READY_FOR_CHECK, GEF_DEAL_DRAFT } = require('../../fixtures');
+const { GEF_DEAL_DRAFT } = require('../../fixtures');
 
 const { BANK1_MAKER1, ADMIN } = MOCK_USERS;
 
 const filters = dashboardFilters;
 
 context('Dashboard Deals filters - filter by status', () => {
-  const ALL_DEALS = [];
+  const ALL_DEALS = [
+    {
+      dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS,
+      status: CONSTANTS.DEALS.DEAL_STATUS.DRAFT,
+      // other deal properties...
+    },
+    {
+      dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS,
+      status: CONSTANTS.DEALS.DEAL_STATUS.READY_FOR_APPROVAL,
+      // other deal statuses...
+    },
+  ];
```

Comment:
Maybe should revert the changes to this file, not sure why we have 2x BSS deals in `ALL_DEALS` which doesn't seem to be used and then we only call `cy.createBssEwcsDeal` once